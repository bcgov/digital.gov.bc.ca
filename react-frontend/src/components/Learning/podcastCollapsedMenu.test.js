import React from 'react';
import renderer from 'react-test-renderer';
import PodcastCollapsedMenu from './podcastCollapsedMenu';

it('renders correctly', () => {
  const tree = renderer.create(<PodcastCollapsedMenu />).toJSON();
  expect(tree).toMatchSnapshot();
});
