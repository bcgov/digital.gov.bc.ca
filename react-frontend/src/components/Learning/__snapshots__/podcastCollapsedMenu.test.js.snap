// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="ant-collapse ant-collapse-icon-position-right"
  role={null}
  style={
    Object {
      "background": "#f2f2f2",
      "border": "none",
    }
  }
>
  <div
    className="ant-collapse-item sc-dkPtyc htfWuT PanelStyled"
  >
    <div
      aria-expanded={false}
      className="ant-collapse-header"
      onClick={[Function]}
      onKeyPress={[Function]}
      role="button"
      tabIndex={0}
    >
      <svg
        aria-hidden="true"
        className="svg-inline--fa fa-chevron-down fa-w-14 ant-collapse-arrow"
        data-icon="chevron-down"
        data-prefix="fas"
        focusable="false"
        role="img"
        style={
          Object {
            "fontSize": "35px",
            "paddingBottom": "6px",
            "paddingTop": "6px",
            "top": "12px",
          }
        }
        viewBox="0 0 448 512"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
          fill="currentColor"
          style={Object {}}
        />
      </svg>
      How to Subscribe
    </div>
  </div>
</div>
`;
